### PR TITLE
Fixing node condition tests.

### PR DESCRIPTION
**Pull request by fubhy**
Those can't work currently without a) enabling the node module and b) actually saving the node type that they create.

### CODE REVIEW

**Comment by fago** [comment]
Why is saving the node type required? Why do the tests pass already then?

**Comment by fubhy** [comment]
The tests don't pass currently. We did not test rules conditions before because of lacking configuraiton for that in the .travis.yml. And once we enabled it, we did not test the node condition for IsPublished (which was the only one back then) because the class was named incorrectly. Hence, no tests were actually firing.

**Comment by fubhy** [comment]
The node type needs to be saved because we get an exception if we try to create a node later on that tries to use that node type. I did not dig deeper.... But it fails :)

File: src/Tests/Condition/NodeIsStickyTest.php [hunk 1]
```diff
@@ -56,11 +49,12 @@ public static function getInfo() {
...
   public function setUp() {
     parent::setup();
-    $entity_manager = $this->container->get('entity.manager');
     $this->conditionManager = $this->container->get('plugin.manager.condition', $this->container->get('container.namespaces'));
-    $this->nodeStorage = $entity_manager->getStorage('node');
-    $this->nodeType = $entity_manager->getStorage('node_type')
-      ->create('node_type', array('type' => 'page'));
+    $this->nodeStorage = $this->container->get('entity.manager')->getStorage('node');
+
+    $this->container->get('entity.manager')->getStorage('node_type')
```
Comment (fago) [inline src/Tests/Condition/NodeIsStickyTest.php]: There should work via $this->entityManager I think.

File: src/Tests/Condition/NodeIsStickyTest.php [hunk 2]
```diff
@@ -56,11 +49,12 @@ public static function getInfo() {
    */
   public function setUp() {
     parent::setup();
-    $entity_manager = $this->container->get('entity.manager');
     $this->conditionManager = $this->container->get('plugin.manager.condition', $this->container->get('container.namespaces'));
-    $this->nodeStorage = $entity_manager->getStorage('node');
-    $this->nodeType = $entity_manager->getStorage('node_type')
-      ->create('node_type', array('type' => 'page'));
+    $this->nodeStorage = $this->container->get('entity.manager')->getStorage('node');
```
Comment (fago) [inline src/Tests/Condition/NodeIsStickyTest.php]: here as well

**Pull request merged**